topdown/crypto.go

```diff
@@ -205,6 +209,55 @@ func builtinCryptoX509ParseRSAPrivateKey(_ BuiltinContext, operands []*ast.Term,
 	return iter(ast.NewTerm(value))
 }
 
+func builtinCryptoParsePrivateKey(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
```

Comment:
(comment about `builtinCryptoX509ParseRSAPrivateKey` function above, not `builtinCryptoParsePrivateKey`)

suggestion: I think that we can rename the implementation function `builtinCryptoX509ParseRSAPrivateKey` to explain that it's returning JWK data. I don't think we can change the built in name, but we can make the implementation clearer at least. We could even consider adding a comment explaining this too since the name is no longer 'ideal'.

Reply:
What about calling it `builtinCryptoJWKPrivateKey ` instead of `builtinCryptoX509ParseRSAPrivateKey`. Would be more accurate, no?

Reply:
Yeah, that's a good name. `builtinCryptoJWKFromPrivateKey` also works.

Reply:
I liked `builtinCryptoJWKFromPrivateKey`, I'll rename it.